recipe/meta.yaml

```diff
@@ -20,33 +20,36 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=39.0.0
+    - setuptools_scm >=3.4.3
     - wheel
   run:
+    # - importlib-metadata <4.3  # flake 8 requires an older version of importlib-metadata
     - autopep8 >=1.6.0,<1.7.0
     - flake8 >=4.0.0,<4.1.0
     - jedi >=0.17.2,<0.19.0
-    - mccabe >=0.6.0,<0.7.0
-    - pluggy
+    - mccabe >=0.6.0,<0.8.0  # flake 8 requires a version less than 0.7.0
```

Comment:
Comments about flake8 should just removed. (This line and the `importlib-metadata` one above.)

Reply:
the unnecessary comments were removed

https://github.com/AnacondaRecipes/python-lsp-server-feedstock/blob/c243edda2252f91f01ca3e414d1d9e6ff454dbc5/recipe/meta.yaml#L27&L31